src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(101);
+        return new Ball(randomColor, randomNumber);
+    }
```

Comment:
let's make color supplier, random - private class-level fields, and 101 - private class-level constant (don't forget to use CONSTANT_CASE, name it smth like BOUND). Good insight for choosing 101 :)
We should avoid static methods, as mentioned in the [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)